_config.yml

```diff
@@ -18,3 +18,30 @@ exclude: [example, packages, tool]
 
 plugins:
   - jekyll-redirect-from
+
+## Site-wide shorthands
+
+alert:
+  tip: >- # Use fa-lightbulb-o for tip?
+    <aside class="alert alert-success" role="alert" markdown="1">
+    <i class="fa fa-check-square-o"></i> **Tip:**
```

Comment:
For tips, do you prefer an `fa-check-square-o` or an `fa-lightbulb-o` icon?

Reply:
I was the one who used lightbulb originally, but am flexible.

Reply:
Any preference? (I have no preference atm.)

Reply:
I've switch it (back) to fa-lightbulb-o.